cachi2/core/package_managers/pip.py

```diff
@@ -1424,6 +1424,19 @@ def _download_dependencies(
             if allow_binary:
                 to_download.extend(w for w in wheels if not w.path.exists())
 
+            if source is None:
+                # at least one wheel exists -> report in the SBOM
+                downloaded.append(
+                    {
+                        "package": req.package,
+                        "version": req.version_specs[0][1],
+                        "kind": req.kind,
+                        "hash_verified": False,
+                        "requirement_file": str(requirements_file.file_path.subpath_from_root),
+                    }
+                )
+                continue
```

Comment:
That's kind of ugly but since there are more stories for cachi2-wheels, which unify handling both distributions, it should be temporary, or not ?

Reply:
Hopefully yes :smile:

Reply:
Should we specifically mention in one of these stories to refactor this?

Reply:
Does @anyone have any suggestions for this?

Reply:
yes we should :slightly_smiling_face: 

probably in the one story for refactoring

Reply:
linked this thread there